enclave-runtime/src/tests.rs

```diff
@@ -402,6 +407,28 @@ fn test_executing_call_updates_account_nonce() {
 	assert_eq!(nonce, 1);
 }
 
+fn test_call_set_update_parentchain_block() {
+	let (_, _, shard, _, _, state_handler) = test_setup();
+	let mut state = state_handler.load_initialized(&shard).unwrap();
+
+	let block_number = 3;
+	let parent_hash = H256::from([1; 32]);
+
+	let header: Header = HeaderT::new(
+		block_number,
+		Default::default(),
+		Default::default(),
+		parent_hash,
+		Default::default(),
+	);
+
+	Stf::update_parentchain_block(&mut state, header.clone()).unwrap();
+
+	assert_eq!(header.hash(), state.execute_with(|| get_parentchain_blockhash().unwrap()));
+	assert_eq!(parent_hash, state.execute_with(|| get_parentchain_parenthash().unwrap()));
+	assert_eq!(block_number, state.execute_with(|| get_parentchain_number().unwrap()));
+}
```

Comment:
Could you move this test to the `stf-executor` crate, where the main change happened? I know we have quite a few tests here, back from the days when we didn't have any components and every test was an integration test. But now that we have separate crates, I think we should also move the tests closer to the actual code. 
It might be a bit of work, since we don't have unit tests setup yet in the `stf-executor`, but I'm happy to help you with that ☺️

Reply:
Can we do that in a separate issue? I'd prefer to merge to PR soon because Litentry is waiting for it.

Reply:
will do in a seperate PR.